src/ucs/sys/sys.c

```diff
@@ -776,9 +780,9 @@ static ucs_status_t ucs_sys_get_memlock_privilege(int *lock_priv)
     UCS_STATIC_ASSERT(CAP_IPC_LOCK < 32);
     status = ucs_sys_get_proc_cap(&ecap);
 
-    *lock_priv = 0;
-    if((status == UCS_OK) && (ecap & UCS_BIT(CAP_IPC_LOCK))) {
-        *lock_priv = 1;
+    *mlock_cap = 0;
```

Comment:
Use `else`

Reply:
why? isn't it simpler to read without `else`?

Reply:
Without `else` you init even if it is not needed..